### ISSUE TITLE
NOTE breaks subtitles in IE11/10

### ISSUE THREAD

**Issue by sampumon** [deestan/srt2vtt#5]
```
NOTE
Converted from .srt via srt2vtt: https://github.com/deestan/srt2vtt
```

Causes `video.textTracks[0].cues` to be empty on IE11; probably IE10 too.

Note that VTT files need also be served as `Content-Type: text/vtt` for subtitles to work on IE11.

**Comment by deestan** [comment]
Sorry for being slow.

Reproduced as described.

The `NOTE` is according to spec, so this is a bug in IE11.  It seems to work if I change the comment from a multi-line comment to a single-line comment, so I'm going to do that.

**Comment by deestan** [comment]
https://github.com/deestan/srt2vtt/commit/50f89f6d1c73e65e6dd85e326149f398717248dd

released on npm as v1.2.1
